scripts/apps/workspace/content/services/ContentService.ts

```diff
@@ -395,12 +395,14 @@ export function ContentService(api, superdesk, templates, desks, packages, archi
                 scope.schema = this.schema(profile, item.type);
                 scope.editor = this.editor(profile, item.type);
                 scope.fields = this.fields(profile);
+                return profile;
             });
         } else {
-            scope.schema = this.schema(null, get(item, 'type', 'text'));
-            scope.editor = this.editor(null, get(item, 'type', 'text'));
-            scope.fields = null;
-            return $q.when();
+            return this.getCustomFields().then(() => {
```

Comment:
Looks like you aren't using the result of `getCustomFields`. How is this supposed to work?

Reply:
it populates `this._fields` which is used by `this.fields` call later